packages/owl-bot/test/scan-googleapis-gen-and-create-pull-requests.ts

```diff
@@ -220,3 +225,45 @@ describe('scanGoogleapisGenAndCreatePullRequests', function () {
     assert.strictEqual(pulls.pulls.length, 1);
   });
 });
+
+describe('copyCodeIntoPullRequest', function () {
+  // These tests use git locally and read and write a lot to the file system,
+  // so a slow file system will slow them down.
+  this.timeout(60000); // 1 minute.
+  const abcRepo = makeAbcRepo();
+  const abcCommits = cmd('git log --format=%H', {cwd: abcRepo})
+    .toString('utf8')
+    .split(/\r?\n/)
+    .map(s => s.trim())
+    .filter(s => s);
+
+  beforeEach(() => {
+    cmd('git checkout main', {cwd: abcRepo});
+  });
+
+  it('copies files into a pull request', async () => {
+    const destRepo = makeDestRepo(bYaml);
+    const pulls = new FakePulls();
+    const issues = new FakeIssues();
+    const octokit = newFakeOctokit(pulls, issues);
+    const factory = newFakeOctokitFactory(octokit, 'test-token');
+    const sourceHash = abcCommits[1];
+
+    await cc.copyCodeIntoPullRequest(
+      abcRepo,
+      sourceHash,
+      destRepo,
+      'test-branch',
+      factory
+    );
+    // Confirm new pull request was pushed to test-branch.
+    const destDir = destRepo.getCloneUrl();
+    const gitLog = cmd('git log test-branch', {cwd: destDir}).toString('utf-8');
+    assert.match(
```

Comment:
I can help add a test for the route in owl-bot.ts too, after we land this; if we go with my suggestion of not passing payload into the handler, then all that test would exercise would be passing the proper parameters into the handler.

Reply:
I'll add the test.

Reply:
Done.